integration-tests/kubernetes/quarkus-standard-way/src/test/resources/kubernetes-with-disabled-manifest-generation.properties

```diff
@@ -0,0 +1,3 @@
+quarkus.kubernetes.enabled=false
+#quarkus.kubernetes.deploy=true
```

Comment:
I envisioned this feature to take precedence over other things but explicitly asking for a Kubernetes deployment doesn't work at this point. This would require changing the logic in `DeploymentUtil` to become aware of the `quarkus.kubernetes.enabled` property, which would be sub-optimal.